### ISSUE TITLE
Make plugin standard lists consistent across repos

### ISSUE THREAD

**Issue by SlnPons** [massalabs/thyra#609]
**Context**
This issue is part of Thyra plan for the mainnet as Milestone 1 to build a robust plugin manager. 
All detail & info can be found [here](https://www.notion.so/massa-innoteam/Improve-polish-key-features-7f828de23fe840bd9d0ced892cb723d2?pvs=4#52642a42ecc44a77a7a5f9dd7679d23b)

--> the objective of this issue is to align the plugin standard lists with agreed standard. These lists exist in the plugin-store repo and thyra repo.

**User flow**
n/a

**How to**

- [x] Check, understand and challenge the standard list defined [here](https://www.notion.so/massa-innoteam/Plug-ins-in-Thyra-2c17c0f694e943c48555bb705fab5453?pvs=4#7bd8a4ac299c41f580aa53c4b568f222).
- [x] Compare and identify differences between the 2 existing plugin standard lists
- [x] Define the manifest that will be associated to the plugins
* note: if you don't know how to make this decision, please reach out to Pierre S. or Gregory L.
- [x] https://github.com/massalabs/thyra-node-manager-plugin/issues/43
- [x] https://github.com/massalabs/thyra-plugin-store/issues/4

**Technical details**
To be added

**QA testing**
TBD

**Comment by SlnPons** [comment]
These issues aren't DoR compliant!!! It is mandatory before you start them. 
Please follow the rules.

**Comment by 0xMazout** [comment]
Manifest of the plugin present in the .zip :
- name 
- author
- description 
- logo 
- home
- version
- apispec

As a remind , this is not the same thing as the data present in the Store

We have the datas in the json of the store , which are used to install the plugin
and we have the manifest present in the .zip of the plugin, which is used to load and use the plugin with the plugin-manager

**Comment by SlnPons** [comment]
@0xMazout @Pierregvx a couple of comments: 
1. Did you make this decision just the 2 of you? 
You should always back-up this type of decision (especially because it's the critical path, aka we won't have time to revisit if we fail, by a senior dev or Greg)
2. To get better in the future: the estimation provided has been exploded. The objective of this estimation is to put it when you start the ticket and try to accomplish within this time :) 
3. @0xMazout I don't understand your comment above "As a remind , this is not the same thing as the data present in the Store"  --> the objective of this ticket is to make sure both repos and so lists of plugins (not installed vs. installed) are consistent (=the same and works the same way). Did I miss sth? 


cc @gregLibert MassaStation now has a manifest for plugins.

**Comment by 0xMazout** [comment]
@SlnPons 
1 ) No as i said in precedent message , Decision was made with @peterjah , it's backed by us, and analyzed to don't create new problems 
2) Ticket was blocked by me we didn't defined a manifest clearly, so "activity time" was not huge on this i think
3 ) It's a misunderstanding , it doesn't need the same stuffs , but in term of User Experience it will be the same , 
my message was to explain to you this is not a goal and it doesn't need to be a goal because they are not for the same usage. 
clear example : store is for install plugins so it needs url to install and it doesn't need description or apispec 

if it's not clear on the 3) lets catchup

**Comment by SlnPons** [comment]
3. is understood but point 2 of this ticket checklist was written to clearly list, identify and share the differences between these 2 standards.... I see this boxe ticked but I haven't seen this info shared clearly. 
Can you do it please? 

This is what I mean by "DoR: have a clear action plan" :)

**Comment by SlnPons** [comment]
@Pierregvx I would have kept the certified logo in the installed plugin list too. 
If a plugin is certified, ie: secure, I potentially want this info to be displayed to end-user anytime. (For instance, when a twitter account is certified, this info is displayed wether you follow it or not)

**Comment by 0xMazout** [comment]
I don't understand.. last time we spoke about the certification we said it was only for Massa Authors plugins.

this have changed ?

**Comment by SlnPons** [comment]
No, all Massa plugins will be certified. Aka a "certified" logo will be display on the front-end.

**Comment by 0xMazout** [comment]
Yes this is what i say it's not a data we need to store. we just have to read the Author name and display it or not . Its a frontend features not related directly to the data and to the lists

**Comment by SlnPons** [comment]
There are 3 points listed on this task, I see 2 out of 3 in done I asked for details as I can't see them anywhere. You give me details I don't understand them so I ask questions.
--> if this is not a standard that should appear on the list: well the job is not done correctly as the list hasn't been challenged and understood as requested by point 1.
--> if this specific "certified logo" is point out as a difference between installed and not installed plugin: I need to understand why because I don't think it's the case. No matter the logic.

This ticket hasn't followed the DoR. It **shouldn't** have been started thus we're late and it creates confusion.
Evil is in the details, we must be very clear and specific. Let's organise a catch up tomorrow. 
Github comments shouldn't be uses for this purpose.

**Comment by Pierregvx** [comment]
Manifest standard :
```json
{
    "author" : "Author  name ",
    "name" : "plugin name ",
    "description" : " a description about the product",
    "logo" : "path to logo",
    "home" : "relative path to home",
    "version" : "X.X.X",
    "apispec":"API specifications"
}
```
